client/gutenberg/extensions/mailchimp/index.js

```diff
@@ -9,32 +9,45 @@ import './editor.scss';
 
 export const name = 'mailchimp';
 
-export const fields = [
-	{ id: 'title', label: __( 'Title' ) },
-	{ id: 'email_placeholder', label: __( 'Placeholder' ) },
-	{ id: 'submit_label', label: __( 'Submit button label' ) },
-	{ id: 'consent_text', label: __( 'Consent text' ) },
-	{ id: 'processing_label', label: __( '"Processing" status message' ) },
-	{ id: 'success_label', label: __( 'Success status message' ) },
-	{ id: 'error_label', label: __( 'Error status message' ) },
-];
-
 export const settings = {
 	title: __( 'MailChimp' ),
 	icon: 'email',
 	category: 'jetpack',
-	keywords: [ __( 'email' ), __( 'mailchimp' ), 'jetpack' ],
-
-	edit,
-	attributes: fields.reduce( ( attrs, field ) => {
-		attrs[ field.id ] = {
+	keywords: [ __( 'email' ), __( 'mailchimp' ) ],
+	attributes: {
+		title: {
 			type: 'string',
-			default: '',
-		};
-		return attrs;
-	}, {} ),
-
-	save: function() {
-		return null;
+			default: __( 'Join my email list' ),
+		},
+		emailPlaceholder: {
+			type: 'string',
+			default: __( 'Enter your email' ),
+		},
+		submitLabel: {
+			type: 'string',
+			default: __( 'Join My Email List' ),
+		},
+		consentText: {
+			type: 'string',
+			default: __(
+				'By clicking submit, you agree to share your email address with the site owner and MailChimp to receive marketing, updates, and other emails from the site owner. Use the unsubscribe link in those emails to opt out at any time.'
+			),
+		},
+		processingLabel: {
+			type: 'string',
+			default: __( 'Processing...' ),
+		},
+		successLabel: {
+			type: 'string',
+			default: __( "Success! You've been added to the list." ),
+		},
+		errorLabel: {
+			type: 'string',
+			default: __(
+				'Oh no! Unfortunately there was an error. Please try reloading this page and adding your email once more.'
+			),
```

Comment:
It's frustrating to duplicate all of these defaults ([#](https://github.com/Automattic/jetpack/pull/11092/files#diff-87c09e43ae364fd1e5472c2476a9d860R104)). Tempting to look for a good way to share data from here to PHP.

Flagging an inconvenience with no apparent solution at this time, not a critique of this PR.